geometry_msgs/PoseArray pose_array
float64[] wait_times